jsk_perception/plugins/nodelet/libjsk_perception.xml

```diff
@@ -290,6 +290,9 @@
          type="jsk_perception::VirtualCameraMono"
          base_class_type="nodelet::Nodelet">
   </class>
+  <class name="jsk_perception/PointPoseExtractor"
+         type="jsk_perception::PointPoseExtractor"
+         base_class_type="nodelet::Nodelet">
```

Comment:
@nakane11 are we missing `</class>` tag?

Reply:
I'm sorry, I overlooked that. 
19d2e95 added the end tag.